### PR TITLE
Improve fair usage message

### PR DESCRIPTION
**Pull request by flvndvd**
## Description

See https://github.com/dust-tt/tasks/issues/749.

This PR improve the fair usage limit message.

FairLimitModal

### CODE REVIEW

File: front/components/app/ReachedLimitPopup.tsx [hunk 1]
```diff
@@ -100,12 +118,13 @@ function getLimitPromptForCode(
...
-          title: "You’ve reach the messages limit",
-          validateLabel: "Check Dust plans",
+          title: "Message quota exceeded",
+          validateLabel: "Ok",
           children: (
             <p className="text-sm font-normal text-element-800">
-              Looks like you've used up all your messages. Check out our paid
-              plans to get unlimited messages.
+              As part of our fair usage policy, we've put a brief pause on your
+              messaging since you've reached the 100 message limit within a 24h
```
Comment (lasryaric) [inline front/components/app/ReachedLimitPopup.tsx:126]: message[s]

**Review by lasryaric — CHANGES_REQUESTED** [review]
The ticket says to redirect to the fair limit usage, not to the subscription page.


Screenshot 2024-05-14 at 15 08 10

**Comment by flvndvd** [comment]
> The ticket says to redirect to the fair limit usage, not to the subscription page.
> 
> Screenshot 2024-05-14 at 15 08 10

Updated, with a GIF in the PR's description.

**Review by lasryaric — APPROVED** [review]
LGTM

**Pull request merged**